.github/CODEOWNERS

```diff
@@ -48,23 +48,41 @@
 
 
 # Angular Migration team
-/core/templates/components/shared-component.module.ts @bansalnitish @srijanreddy98
-/core/templates/services/loader.service*.ts @bansalnitish @srijanreddy98
-/core/templates/components/material.module.ts @bansalnitish @srijanreddy98
-/core/templates/components/oppia-angular-root.component.ts @bansalnitish @srijanreddy98
-/core/templates/base-components/oppia-root.directive.ts @bansalnitish @srijanreddy98
-/core/templates/base-components/oppia-root.directive.html @bansalnitish @srijanreddy98
-/core/templates/filters/translate.pipe.ts @bansalnitish @srijanreddy98
-/core/templates/services/angular-services.index.ts @bansalnitish @srijanreddy98
-/core/templates/services/contextual/logger.service.ts @bansalnitish @srijanreddy98
-/core/templates/services/contextual/logger.service.spec.ts @bansalnitish @srijanreddy98
-/core/templates/services/i18n-language-code.service.ts @bansalnitish @srijanreddy98
-/core/templates/services/loader.service*.ts @bansalnitish @srijanreddy98
-/core/templates/services/question-validation.service*.ts @bansalnitish @srijanreddy98
-/core/templates/services/request-interceptor.service*.ts @bansalnitish @srijanreddy98
-/core/templates/services/translate.service.ts @bansalnitish @srijanreddy98
-/core/templates/services/translations-backend-api.service.ts @bansalnitish @srijanreddy98
-/core/templates/services/UpgradedServices.ts @bansalnitish @srijanreddy98
+# TODO(10619): Add @bansalnitish as owner after 2020-09-26
```

Comment:
Should we just have one TODO comment on top?

Reply:
Done!